ch6/errdefer.zig

```diff
@@ -0,0 +1,17 @@
+const std = @import("std");
```

Comment:
Maybe this example could be easy-runnable, like, you haven't touched allocators so far! It'll be important for the readers an example they can run/create the main with the knowledge they already have from the previous chapters

Reply:
added the same comment in the review. Not sure if allocators were already a topic in the previous chapters. If not, this example could be something more straightforward

Reply:
Yeap, I've read the previous chapters and they don't have anything touching allocators

Reply:
It makes a lot of sense.